tests/example-success/expected_results.json

```diff
@@ -1,4 +1,52 @@
 {
-  "version": 1,
-  "status": "pass"
-}
+    "message": null,
+    "status": "fail",
```

Comment:
The status should be `pass`, as all tests are passing.

Reply:
Good catch, sorry about that!